contents/blog/features-sell.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2022-10-20
+title: "Is this the death of benefits selling?"
```

Comment:
On first glance, I don't really know what "benefits selling" means. Some ideas for alternatives:
- The death of selling benefits
- The death of selling based on "why"
- Your "why" doesn't sell anymore
- I don't want to be sold by your "why"